headless/src/jsMain/kotlin/dev/fritz2/headless/components/toast.kt

```diff
@@ -50,7 +50,7 @@ fun <E : HTMLElement> toastContainer(
     scope: (ScopeContext.() -> Unit) = {},
     tag: TagFactory<Tag<E>>
 ) {
-    PortalRenderContext.portal(classes, id, scope, tag) {
+    PortalRenderContext.portal(classes, id, scope + PortalRenderContext.scopeContext, tag) {
```

Comment:
`scope + other` would override values from `scope` by values from `other` with the same key.
Wouldn't you expect the scope-values provided by the caller of `toastContainer` to "win"?

If so, the sequence of both operands must be switched.

If not, I think this behaviour should at least be documented then, as it behaves differently than in the rest of the framework.